content/en/docs/pipelines/multi-user.md

```diff
@@ -77,11 +77,11 @@ print(client.list_runs(namespace='<Your namespace>'))
 To store your user namespace as the default context, use the
 [`set_user_namespace`](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.client.html#kfp.Client.set_user_namespace)
 method. This method stores your user namespace in a configuration file at
-`$HOME/.config/kfp/context.json`. After setting a default namespace, the SDK
+`$HOME/.config/kfp/credentials.json`. After setting a default namespace, the SDK
 methods default to use this namespace if no namespace argument is provided.
 
 ```python
-# Note, this saves the namespace in `$HOME/.config/kfp/context.json`. Therefore,
+# Note, this saves the namespace in `$HOME/.config/kfp/credentials.json`. Therefore,
```

Comment:
I think this is right: https://github.com/kubeflow/pipelines/blob/e3992faf8305570ca56c5932d701eb8765305568/sdk/python/kfp/_client.py#L118